### ISSUE TITLE
Doesn't work VaapiPostProcessScaler

### ISSUE THREAD

**Issue by AlexanderKomarov** [01org/libyami#817]
VaapiPostProcessScaler doesn't work. It returns SharedPtr < VideoFrame >  with zeros in crop.width and crop.height. I tested it on the vpp test and my application. I use DRM as display.
But all tests are passed.

I use Ubuntu 16.04. The latest libva, libyami and intel-vaapi-driver.

**Comment by xuguangxin** [comment]
could you share you command line and try our yamivpp application https://github.com/01org/libyami-utils/blob/master/tests/vpp.cpp like  "yamivpp <option> input_1920x1080.i420 output_320x240.yv12"

**Comment by AlexanderKomarov** [comment]
I just run vpp/unittest. Also I added this line `std::cout << std::to_string(dest->crop.width) << " " << std::to_string(dest->crop.height) << std::endl;` to VppDeinterlace function in vaapipostprocess_scaler_unittest.cpp after this line `EXPECT_EQ(YAMI_SUCCESS, scaler.process(src, dest));`
And I get only zeros. Is it normal?

I ran our yamivpp application. And it works.

**Comment by xuguangxin** [comment]
Yes, it's defined feature. If crop is zero, It will scale to the entire surface.
We have src crop and dest crop. src crop means you only need a sub rect on the source,  dest crop means you only need scale src to the  sub rect of dest.

**Comment by AlexanderKomarov** [comment]
ok, thank you for the answer.

Also quick question I tried to test YAMI_FOURCC_Y800 format for scaling but I get the error: "the requested function is not implemented". Do you plan to support it? And where can I get the list of supported formats for scaling?

**Comment by xuguangxin** [comment]
It's the error returned by libva. we depend on libva to do vpp.

Hi @xhaihao,
Can we query csc format in libva?

HI @AlexanderKomarov 
Could you share us the usecase? Why you need Y800 scaling?

thanks

**Comment by xhaihao** [comment]
vaQuerySurfaceAttributes can return the supported surface attribute list for a given config, including pixel formats

**Comment by AlexanderKomarov** [comment]
@xuguangxin @xhaihao thanks for the answers.

We try to use Intel GPU for decoding/encoding JPEG from camera, scaling and transforming images. We use gray image in our system. Our system does face detection and face recognition.

**Comment by AlexanderKomarov** [comment]
@xuguangxin @xhaihao maybe is there some sense to add the issue of adding support Y800 for vpp in libva project?

**Comment by xuguangxin** [comment]
@xhaihao , what's the suggestion?:)
